src/evaluation_system/model/solr_core.py

```diff
@@ -151,7 +151,7 @@ def create(
         :param data_dir: Data directory for this core (if left unset, a local "data" directory in instance_dir will be used)
         :param config: The configuration file (expected in instance_dir/conf)
         :param schema: The schema file (expected in instance_dir/conf)
-        :param check_if_exist: check for the existence of the instance directorie
+        :param check_if_exist: check for the existence of the instance directory
         """
```

Comment:
This point by codecov should be addressed

src/evaluation_system/model/solr_core.py#L154

Added line #L154 was not covered by tests

Reply:
okey, I will check how to add that line